app/routes.js

```diff
@@ -246,8 +250,14 @@ module.exports = (app, passport, mongoose) => {
           });
       });
 
-    app.get('/getStudentResumes', (req, res) => {
-        if (Object.keys(req.body)[0] === null) {
+    // gets Tags
+    app.get('/getTags',  (req, res) => {
```

Comment:
When you're building an API, you don't have the HTTP verb in the route name. In this case, you should have the following routes relating to tags:
- `GET /tags` to get the tags
- `POST(/PUT) /tags` to set the tags